kitura-api.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const tools = require('./helper');
+
+const gitURL = 'https://github.com/IBM-Swift/generator-swiftserver-projects';
+const gitBranch = 'openAPI';
+
+let args = process.argv.slice(2);
+
+if (args.length > 0) {
+    if (args.indexOf('--help') > -1) {
+        printHelp();
+        process.exit(0);
+    }
+}
+
+let currentDirPath = path.resolve("./");
```

Comment:
This section of the file is common between `init`, `basic` and `api`, can we have more common code so we don't duplicate it?